Makefile

```diff
@@ -185,7 +185,7 @@ minikube-load-all: $(addprefix minikube-load-,$(TARGETS))
 
 # Build chaosli
 chaosli:
-	GOOS=darwin GOARCH=${GOARCH} CGO_ENABLED=0 go build -ldflags="-X github.com/DataDog/chaos-controller/cli/chaosli/cmd.Version=$(VERSION)" -o bin/chaosli/chaosli_darwin_${GOARCH} ./cli/chaosli/
+	GOOS=darwin GOARCH=$(GOARCH) CGO_ENABLED=0 go build -ldflags="-X github.com/DataDog/chaos-controller/cli/chaosli/cmd.Version=$(VERSION)" -o bin/chaosli/chaosli_darwin_$(GOARCH) ./cli/chaosli/
```

Comment:
Why do you say it's wrong?
https://www.gnu.org/software/make/manual/html_node/Reference.html#Reference

Did anything is not working with the previous syntax?